__init__.py

```diff
@@ -133,6 +134,7 @@
     from . import icons
     from . import image_utils
     from . import overrides
+    from . import override_extension_draw
```

Comment:
This needs to be secured from execution for Blender < 4.2.0. Otherwise these version will not work as the tests also show.

Reply:
done